portfolio/src/main/webapp/script.js

```diff
@@ -47,9 +48,18 @@ function createCommentElement(comment) {
 
   const authorElement = document.createElement('span');
   authorElement.innerText = comment.author + "\n";
-
   
   commentElement.appendChild(authorElement);
   commentElement.appendChild(bodyElement);
   return commentElement;
+}
+
+
+/** deletes all comments */
+function deleteEverything(){
+  const request = new Request('/delete-data', {method: 'POST'});
+  fetch(request).then(response => response.json()).then(() => {
+    getMessages();
+  });
+  getMessages();
```

Comment:
What is this line for?

Reply:
An extra line left over from testing, now removed.